versions.props

```diff
@@ -2,6 +2,8 @@ com.netflix.nebula:nebula-dependency-recommender = 7.5.6
 com.netflix.nebula:nebula-test = 7.2.5
 com.google.guava:guava = 23.6.1-jre
 org.immutables:value = 2.7.5
+org.junit.jupiter:* = 5.4.0
```

Comment:
could you use the bom instead?

Reply:
Also 5.4.2? https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter